liesel/distributions/mvn_degen.py

```diff
@@ -95,8 +103,37 @@ class MultivariateNormalDegenerate(tfd.Distribution):
       operation and can be avoided by specifying the corresponding arguments.
     * When you draw samples from the distribution via :meth:`.sample`, it is always
       necessary to compute the eigendecomposition of the distribution's precision
-      matrices once and cache it, because sampling requires both the eigenvalues
-      and eigenvectors.
+      matrices once and cache it, because sampling requires both the eigenvalues and
+      eigenvectors.
+
+    **Details on sampling**
+
+    If we have a singular precision matrix :math:`P`, we can view it as the
+    pseudo-inverse :math:`\\Sigma^+` of the variance-covariance matrix :math:`\\Sigma`.
+    We can obtain :math:`\\Sigma` by finding the singular value decomposition
+
+    .. math::
+        \\Sigma^+ = QA^+Q^T,
+
+    where :math:`Q` is the orthogonal matrix of eigenvectors and :math:`A^+` is the
+    diagonal matrix of singular values of :math:`\\Sigma^+ = P`. If the precision matrix
+    is singular, then :math:`\\text{diag}(A^+)` contains zeroes. Now we take the inverse
+    of the non-zero entries of :math:`\\text{diag}(A^+)`, while the zero entries remain
+    at zero, resulting in a matrix :math:`A`. We can now write
+
+    .. math::
+        \\Sigma = Q A Q^T.
+
+    To sample from :math:`N(\\mu, \\Sigma)`, we first draw a sample
```

Comment:
$N(\mu, \Sigma)$ is not a proper normal distribution since Sigma is singular.  Maybe you can call it $N_D$ or $N^\perp$.  It's probably easiest to follow Rue and Helds notation.

Reply:
Rue and Held seem to avoid using a specific Notation for a singular normal distribution. Thomas uses the same notation as for a non-singular normal distribution in the appendix of his book, (B.3.2). I might just rephrase the sentence to avoid the symbol, too.